debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/MongoDbReplicaSet.java

```diff
@@ -69,6 +84,20 @@ public static class Builder {
         private PortResolver portResolver = new RandomPortResolver();
         private boolean skipDockerDesktopLogWarning = false;
         private DockerImageName imageName;
+        private boolean authEnabled;
+        private String rootUser = "root";
+        private String rootPassword = "secret";
```

Comment:
nitpick: I'd define `root` and `secret` as the constants as they are used on multiple places